arwen/contexts/output.go

```diff
@@ -110,11 +110,20 @@ func (context *outputContext) CensorVMOutput() {
 	context.outputState.Logs = make([]*vmcommon.LogEntry, 0)
 }
 
+// ResetConsumedGas will set to 0 all gas used from output accounts, in order
+// to properly calculate the actual used gas of one smart contract when called in sync
+func (context *outputContext) ResetConsumedGas() {
```

Comment:
`ResetGas`? As not only used gas is reset, but also the gas limit?

Reply:
no need to reset gaslimit - deleted.